.github/workflows/ci.yaml

```diff
@@ -143,6 +143,7 @@ jobs:
           -DCMAKE_TOOLCHAIN_FILE=${{ env.VCPKG_DEST_WIN }}\scripts\buildsystems\vcpkg.cmake ^
           -DVCPKG_TARGET_TRIPLET=x64-windows-static-release ^
           -DVERBOSE_CONFIGURE=ON ^
+          -DMSVC_RUNTIME_DYNAMIC=OFF ^
```

Comment:
I'm confused, what do we want here? Making the built and uploaded artifact useful to someone? Or we just upload it for fun?
If it is the former it make sense to have static runtime. If it is the latter, why upload it at all?